content/guides/virtualizing/simnow.md

```diff
@@ -5,42 +5,104 @@ date = "2016-12-23T06:09:57.000Z"
+
+Choose a location to save the hard disk image to.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image7.png)
+
+Assign a size to your hard disk image and click **OK**.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image8.png)
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image9.png)
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image10.png)
+
+Click **OK** to complete the process, then exit disktool.
 
 ##### Mounting ISO image for installation
 
-![](/files/images/simnow_image11.png) Now, run _"simnow.exe"_ from the installation directory to run SimNow. ![](/files/images/simnow_image12.png) Click on the Open BSD icon. ![](/files/images/simnow_image13.png) Choose _"cheetah_1p.bsd"_ from the SimNow directory. ![](/files/images/simnow_image14.png) After the BSD is loaded, click on File > Set IDE Primary Master Image ![](/files/images/simnow_image15.png) Browse to the hard disk image that you just created in the previous part. ![](/files/images/simnow_image16.png) Now, click File > Set IDE Secondary Master Image. ![](/files/images/simnow_image17.png) Change File type to _"ISO Images (*.iso)"_ and browse to the downloaded Haiku iso image.
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image11.png)
+
+Now, run _"simnow.exe"_ from the installation directory to run SimNow.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image12.png)
+
+Click on the Open BSD icon.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image13.png)
+
+Choose _"cheetah\_1p.bsd"_ from the SimNow directory.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image14.png)
+
+After the BSD is loaded, click on **File** > **Set IDE Primary Master Image**.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image15.png)
+
+Browse to the hard disk image that you just created in the previous part.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image16.png)
+
+Now, click **File** > **Set IDE Secondary Master Image**.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image17.png)
+
+Change File type to _"ISO Images (*.iso)"_ and browse to the downloaded Haiku iso image.
 
 ##### Using the Anyboot image
 
-You can also use the anyboot image to directly run Haiku without installation in the VM. ![](/files/images/simnow_image20.png) Just choose the anyboot image as your IDE Primary Master, and you're ready to go!
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image20.png)
+
+You can also use the anyboot image to directly run Haiku without installation in the VM. Just choose the anyboot image as your IDE Primary Master, and you're ready to go!
```

Comment:
Remove the "also". The screenshot shows the stone old alpha and the filename has a .image suffix, probably forcing you to filter for *.* (any file).
Today's Haiku images are named "haiku-nightly-anyboot.iso". While having the "nightly" in there isn't totally future-proof (once we have another release), but beats the old alpha naming. Just rename the image for the screenshot.

Reply:
I don't know if I can recreate this screenshot. Should we add a note or I use photo editing software?

Reply:
I think I'd better follow this guide and replace all the old screenshots.

Reply:
I agree. Thanks!